backend/database/migrations/2018_02_22_023241_create_transactions_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTransactionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('transactions', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('card_id')->unsigned();
+            $table->foreign('card_id')->references('id')->on('cards');
+            $table->integer('user_id')->unsigned();
+            $table->foreign('user_id')->references('id')->on('users');
+            $table->time('transaction_time');
```

Comment:
Lets use laravel's timestamps feature so we get automated timestamps with additional info.

Reply:
(that would be`$table->timestamps();`)